src/elements/MenuDrawer/MainMenu/MainMenu.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { Text, View } from 'react-native';
+
+import { Icon } from '@elements';
+import { IconName } from '@elements/Icon';
+import * as colors from '@util/colors';
+
+import styles from './MainMenu.styles';
+
+const MainMenu = props => {
+	let menuStyle = {};
+	if (props.text === 'Contact Us') {
+		menuStyle = {
+			...styles.mainMenu,
+			marginBottom: 50,
```

Comment:
Not really an issue, but perhaps we could stick these styles in an object in the styles file and spread them alongside the `mainMenu` styles? 
e.g.
```
menuStyle = {
    ...styles.mainMenu,
    ...styles.contactUsPadding,
}
```

Just a thought

Reply:
Good idea -- to keep everything in one place.